lib/index.js

```diff
@@ -1723,38 +1749,35 @@ exposeMethod('pluck');
  */
 
 Stream.prototype.pickBy = function (f) {
-
-    return this.consume(function (err, x, push, next) {
+    return this.map(objectOnly(function (x) {
         var out = {};
-        if (err) {
-            push(err);
-            next();
+        var seen = {};  // prevents testing overridden properties multiple times.
```

Comment:
What's the best way to deal with this? Put the `Object.create` behind a `if (es5)` check? Use strict `seen[prop] === true` instead of `prop in seen`? None of the `Object.prototype` properties `=== true`, right?